olive/systems/docker/utils.py

```diff
@@ -58,23 +59,23 @@ def create_metric_volumes_list(metrics: List[Metric], container_root_path: Path,
     for metric in metrics:
         metric_path = container_root_path / "metrics" / metric.name
         if metric.user_config.user_script:
-            user_script_path = str(Path(metric.user_config.user_script).resolve())
-            user_script_name = Path(metric.user_config.user_script).name
+            user_script_path = get_local_path(metric.user_config.user_script)
```

Comment:
>get_local_path [](http://example.com/codeflow?start=31&length=14)

no need call get_local_path for user_script and script_dir. Only need for data_dir